bin/edge_cases/knapsack_pro_queue_rspec_in_background

```diff
@@ -10,11 +10,21 @@ export KNAPSACK_PRO_CI_NODE_TOTAL=2
 export KNAPSACK_PRO_BRANCH=test-1
 export KNAPSACK_PRO_COMMIT_HASH=commit-${1:-1}
 #export KNAPSACK_PRO_TEST_FILE_PATTERN="spec/controllers/**/*_spec.rb"
-#export KNAPSACK_PRO_RSPEC_SPLIT_BY_TEST_EXAMPLES=true
+#export KNAPSACK_PRO_TEST_FILE_PATTERN="spec/controllers/articles_controller_spec.rb"
```

Comment:
@shadre you can uncomment this line to only run 2 test files. Articles controller and test file defined in KNAPSACK_PRO_SLOW_TEST_FILE_PATTERN

You can run tests in terminal

```
bin/edge_cases/knapsack_pro_queue_rspec_in_background
```